mmv1/third_party/terraform/services/integrations/resource_integrations_auth_config_test.go

```diff
@@ -0,0 +1,130 @@
+package integrations_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccIntegrationsAuthConfig_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"client":        acctest.BootstrapIntegrationsClient(t, "us-east4").Name,
```

Comment:
I don't think we ever use this value as-written. Clients that are used by the provider are created in generated provider code, not in the test code. What are you trying to do here?

Reply:
We need to ensure that client is provisioned before any auth config tests hence added this bootstrapping to ensure the same.